webpack.config.js

```diff
@@ -44,14 +44,18 @@ module.exports = {
       exclude: /node_modules/,
       loader: ExtractTextPlugin.extract(
         'style-loader',
-        'css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]&sourceMap!postcss-loader' // eslint-disable-line
+        'css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]&sourceMap!postcss-loader' // eslint-disable-line max-len
```

Comment:
I think this webpack file might worth an update. THe boierplate has changed a bit